### DISCUSSION TITLE
Environment distinction

### DISCUSSION THREAD

**Discussion by JarnDev** [magicbell/community#117] [Questions] [answered]
Hi there, I realized that there is no way to separate the app usage on environments (dev, stage, prod), if a user shares the identifier on all environments if a notification is sent on dev, it will be shown on all other environments on my app... 

I saw that I can create more projects, should I use that approach? Should I Create a project for each environment that I use? If so how many projects am I allowed to create?

My initial approach was to append the environment string onto the external_id like

dev => dev-[external_id]
stage => stage-[external_id]
prod => external_id

But if I can create projects for each environment it would be way better!

**Comment by JarnDev** [comment, marked as answer]
Used Projects to separate Environments!

**Reply by smeijer** [reply]
Hi @JarnDev, sorry for the late reply. Using projects as environments is exactly what I would have recommended. You're not being charged for additional projects, so please feel free to create as many projects in your workspace as you need.
